docs/manual/creating-a-checker.tex

```diff
@@ -1937,17 +1938,29 @@
 \begin{Verbatim}
 java -cp $CHECKERFRAMEWORK/checker/dist/checker.jar \
   org.checkerframework.dataflow.cfg.CFGVisualizeLauncher \
-  MyClass.java output/ -class MyClass -method test -pdf
+  MyClass.java -output output/ -class MyClass -method test -pdf
```

Comment:
To simplify the command, how about leaving off the `-output` command-line argument?